README.rst

```diff
@@ -41,6 +41,10 @@ directory and execute the command::
 
     python setup.py install
 
+to have known to work dependencies use beforehand::
```

Comment:
I would recommend putting these before `python setup.py install`. I know you mentioned that we need to run this beforehand, but I feel people have a tendency of following commands in sequence of the way they're written in the readme.

Reply:
yeah, I agree